tree/ntupleutil/v7/inc/ROOT/RNTupleImporter.hxx

```diff
@@ -254,10 +257,16 @@ public:
    /// Directly uses the provided tree and opens the output file for writing (update).
    static RResult<std::unique_ptr<RNTupleImporter>> Create(TTree *sourceTree, std::string_view destFileName);
 
+   /// Imports all TTree objects present in `sourceFile` as RNTuple objects to `destFile`.
+   static RResult<std::unique_ptr<RNTupleImporter>> Create(std::string_view sourceFile, std::string_view destFile);
+
    RNTupleWriteOptions GetWriteOptions() const { return fWriteOptions; }
    void SetWriteOptions(RNTupleWriteOptions options) { fWriteOptions = options; }
-   void SetNTupleName(const std::string &name) { fNTupleName = name; }
-   void SetMaxEntries(std::uint64_t maxEntries) { fMaxEntries = maxEntries; };
+   void SetMaxEntries(std::uint64_t maxEntries) { fMaxEntries = maxEntries; }
+
+   /// Throws an exception in case multiple TTrees are imported simultaneously.
+   ROOT::Experimental::RResult<void> SetNTupleName(const std::string &ntupleName);
```

Comment:
I think we should use string views consistently, here in and in the other overload.